packages/browsers/src/httpUtil.ts

```diff
@@ -48,7 +48,7 @@ export function httpRequest(
     protocol: url.protocol,
     hostname: url.hostname,
     port: url.port,
-    path: url.pathname,
+    path: url.pathname + url.search,
```

Comment:
Nit: I think this should be `url.href` as that is the full url
```suggestion
    path: url.href,
```